sections/round_structure.tex

```diff
@@ -26,6 +26,13 @@
 
 \vspace*{\fill}
 
-\includegraphics[width=1.8\linewidth]{\art/minotaur.png}
+\includegraphics[width=1.6\linewidth]{\art/minotaur.png}
+
+\vspace*{\fill}
+
+\note{6}{
+  In the Alliance game, teams take turns alternately.
+  At the start of your team's first turn of the Round, you can choose who of your team goes first and who last in that Round.
```

Comment:
```suggestion
  At the start of your team's first turn of the Round, you can choose which of the team members goes first in that Round.
```

Reply:
+1 for the suggestion, the "last" part is kind of clear if you specify the one going first and there are two players on a team.

Reply:
Czech translation for that
> Ve hře Alliance se týmy po každém tahu střídají. Na začátku prvního tahu vašeho týmu v daném kole si můžete vybrat, který z členů týmu půjde v tomto kole první.

Reply:
```suggestion
  At the start of your team's first turn of the Round, choose who goes first and last in that Round.
```
Alternative version if we want to keep the "last" word.